lib/src/rules/prefer_iterable_whereType.dart

```diff
@@ -81,7 +81,7 @@ class _Visitor extends SimpleAstVisitor<void> {
       if (expression is IsExpression && expression.notOperator == null) {
         final target = expression.expression;
         if (target is SimpleIdentifier &&
-            target.name == arg.parameters!.parameters.first.identifier!.name) {
+            target.name == arg.parameters?.parameters.first.identifier!.name) {
```

Comment:
Consider assigning `arg.parameters?.parameters.first.identifier` to a local variable to remove the other null check operator.

Reply:
Great catch.  I missed that!